sql/item_weapon.sql

```diff
@@ -3975,12 +3975,54 @@ INSERT INTO `item_weapon` VALUES (21056,'tomonari',10,0,0,0,0,2,1,420,118,0);
 INSERT INTO `item_weapon` VALUES (21057,'tomonari_+1',10,0,0,0,0,2,1,407,119,0);
 INSERT INTO `item_weapon` VALUES (21058,'shichishito',10,0,54,54,42,2,1,450,126,0);
 INSERT INTO `item_weapon` VALUES (21059,'ichijinto',10,0,0,0,0,2,1,420,121,0);
+INSERT INTO `item_weapon` VALUES (21060,'mjollnir',11,0,242,242,228,3,1,308,160,0);
 INSERT INTO `item_weapon` VALUES (21061,'mjollnir',11,0,242,242,228,3,1,308,160,0);
-INSERT INTO `item_weapon` VALUES (21062,'yagrush',11,0,242,242,228,3,2,267,133,0);
-INSERT INTO `item_weapon` VALUES (21063,'yagrush',11,0,242,242,228,3,2,267,133,0);
+INSERT INTO `item_weapon` VALUES (21062,'yagrush',11,0,242,242,228,3,3,267,133,0);
```

Comment:
Okay then I'll have to go over all of them when I get a chance unless
someone corrects them first

On Saturday, April 2, 2016, TeoTwawki <notifications@github.com> wrote:

> In sql/item_weapon.sql
> <https://github.com/DarkstarProject/darkstar/pull/2970#discussion_r58294077>
> :
>
> >  INSERT INTO `item_weapon` VALUES (21061,'mjollnir',11,0,242,242,228,3,1,308,160,0);
> > -INSERT INTO `item_weapon` VALUES (21062,'yagrush',11,0,242,242,228,3,2,267,133,0);
> > -INSERT INTO `item_weapon` VALUES (21063,'yagrush',11,0,242,242,228,3,2,267,133,0);
> > +INSERT INTO `item_weapon` VALUES (21062,'yagrush',11,0,242,242,228,3,3,267,133,0);
>
> All of them are wrong, both before and after you touched them.
>
> That isn't part of the weapon. Look at the item help-text:
>
> [image: Yagrush]
> <https://camo.githubusercontent.com/dece6f6acc6677656ecf38ecb12c697dc6948083/687474703a2f2f7669676e65747465312e77696b69612e6e6f636f6f6b69652e6e65742f666678692f696d616765732f322f32362f5961677275736839392e706e672f7265766973696f6e2f6c61746573743f63623d3230313230373133303134373130>
>
> That is in the aftermath effects, not the items basic stats.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/DarkstarProject/darkstar/pull/2970/files/fd946bedc85a9b341ff22779b129fe16915d51cf#r58294077>
>